lib/bgs/services/vet_record.rb

```diff
@@ -21,12 +21,13 @@ def bean_name
     # with the first being the first character of the poa code
     # and the second field the remaining 2 characters
     # As per BGS, CLAIM NUMBER and ssn are synonymous
```

Comment:
We should probably delete this comment since we've discovered it's not always true.

Reply:
Reworded a bit [here](https://github.com/department-of-veterans-affairs/bgs-ext/pull/73/commits/4eb4fc88eee39cc61edb8001101ea127b55e5ff2).